### PR TITLE
13 show mock data favourites

### PR DESCRIPTION
**Pull request by JustinDotMorris**
Completed three favourites text displayed on screen & testing completed and passed.

### CODE REVIEW

File: src/components/FavouritesCard/FavouritesCard.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import "./FavouritesCard.scss";
+
+const FavouritesCard = () => {
+  const array_fav = [
```
Comment (DanForder) [inline src/components/FavouritesCard/FavouritesCard.jsx:4]: use `camelCase` for variables

File: src/components/FavouritesCard/FavouritesCard.test.jsx [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import FavouritesCard from "./FavouritesCard";
+import { customRender } from "../../utils/testUtils";
+
+it("should render the text on the display", () => {
+  const { container } = customRender(
+    <FavouritesCard>
+      <p>Hello there</p>
+    </FavouritesCard>
```
Comment (DanForder) [inline src/components/FavouritesCard/FavouritesCard.test.jsx:8]: The `<FavouritesCard>` isn't used in quite the same way as the `<Layout>`, you don't need a `p` tag inside it. Simply `<FavouritesCard/>` should do!

File: src/pages/Favourites/Favourites.jsx [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import "./Favourites.scss";
+import FavouritesCard from "../../components/FavouritesCard/FavouritesCard";
+import Layout from "../../components/Layout/Layout";
+// import { useEffect } from "react";
```
Comment (DanForder) [inline src/pages/Favourites/Favourites.jsx:4]: remove commented code

**Review by DanForder — CHANGES_REQUESTED** [review]
Remove the two empty SCSS files:
- FavouritesCard.scss
- Favourites.scss

**Review by DanForder — CHANGES_REQUESTED** [review]
remove 2 empty scss files

**Review by DanForder — APPROVED** [review]

**Review by DanForder — APPROVED** [review]

**Review by katherinepv — APPROVED** [review]

**Pull request merged**